web/img/SuggererUnRestaurant.svg

```diff
@@ -0,0 +1,160 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Please use dashes (`-`) in filenames, and English

`web/img/SuggererUnRestaurant.svg` → `web/img/suggest-restaurant.svg`